.testcoverage.yml

```diff
@@ -37,7 +37,7 @@ override:
     path: ^internal/resources/selfmonitor$
   - threshold: 75
     path: ^internal/resourcelock$
-  - threshold: 86
+  - threshold: 85
```

Comment:
🤔

Reply:
Was too lazy to add more tests 🤣

Reply:
No worries, we were just joking, because you are usually the one pushing higher coverage thresholds 😂